code/__DEFINES/is_helpers.dm

```diff
@@ -1,13 +1,11 @@
 // simple is_type and similar inline helpers
 
-#define islist(L) (istype(L, /list))
-
 #define in_range(source, user) (get_dist(source, user) <= 1 && (get_step(source, 0)?:z) == (get_step(user, 0)?:z))
 
-#define ismovableatom(A) (istype(A, /atom/movable))
-
 #define isatom(A) (isloc(A))
 
+#define ismovableatom(A) (ismovable(A)) // we got ismovable in 513
```

Comment:
Why not just do a find & replace?

Reply:
increasing our diff. we can do this if TG also does it.